.github/workflows/test.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         - 10
```

Comment:
Should we also remove 10, as it has reached end of life?

Reply:
I was thinking that, but it'd technically be a breaking change (and one that we don't really need to make right now). We should probably roll it up into the next breaking change we make, though.

Reply:
Although actually I forgot we hadn't 1.0'd this repo yet, so yeah actually let's do that now and roll it up into the v1.0 release.

Reply:
That would be sweet